demisto_sdk/commands/format/update_pythonfile.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from typing import Tuple
+
+from demisto_sdk.commands.common.tools import LOG_COLORS, print_color
+from demisto_sdk.commands.format.format_constants import (
+    ERROR_RETURN_CODE, SKIP_VALIDATE_PY_RETURN_CODE, SUCCESS_RETURN_CODE)
+from demisto_sdk.commands.format.update_generic_yml import BaseUpdate
+
+BLACK_INTERNAL_ERROR = 123
+
+
+class PythonFileFormat(BaseUpdate):
+    """PythonFileFormat class is designed to update python file according to Demisto's convention.
+
+        Attributes:
+            input (str): the path to the file we are updating at the moment.
+            output (str): the desired file name to save the updated version of the python file to.
+    """
+
+    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '',
+                 no_validate: bool = False):
+        super().__init__(input, output, path, from_version, no_validate)
+        self.no_validate = True
```

Comment:
`no_validate` has a default value of `False`, which you pass to the base class and then override it.
this is confusing, add a comment on why you force it to be `True`.
or change the default to `True` and remove the override.

Reply:
done